spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe "Food Index" do 
+  it "has the top 10 food items that contain this ingredient", :vcr do 
+    visit root_path 
+    # fill_in "Ingredient Search", with: 'sweet potatoes' #getting capybara disabled error
```

Comment:
Try this syntax: `fill_in :q, with: "sweet potatoes"` --> the form field is Q rather than Ingredient search

Reply:
If that doesn't work, I'd love to see the error you're getting!